grammar/SignalFlowV2Parser.g4

```diff
@@ -0,0 +1,143 @@
+/**
+ * Copyright (C) 2016 SignalFx, Inc.
+ */
+parser grammar SignalFlowV2Parser;
+
+options { tokenVocab = SignalFlowV2Lexer; }
+tokens { INDENT, DEDENT }
+
+program
+  : ( NEWLINE | statement )* EOF
+  ;
+
+functionDefinition
+ : DEF ID parameters ':' suite
+ ;
+
+parameters
+ : OPEN_PAREN varArgsList? CLOSE_PAREN
+ ;
+
+varArgsList
+ : varArgsListParamDef ( ',' varArgsListParamDef )*
```

Comment:
You only need to make it a token if you
a) need to get the token that was used programatically (like a '+' or a '-' in a expr
b) need to group multiple characters into one token